packages/integrations/netlify/package.json

```diff
@@ -30,7 +30,7 @@
     "build:ci": "astro-scripts build \"src/**/*.ts\"",
     "dev": "astro-scripts dev \"src/**/*.ts\"",
     "test-fn": "mocha --exit --timeout 20000 test/functions/",
-    "test-edge": "deno test --allow-run --allow-read --allow-net ./test/edge-functions/",
+    "test-edge": "deno test --allow-run --allow-read --allow-net --allow-env ./test/edge-functions/",
```

Comment:
This addition is needed for the test to run, as the adapter executes `Deno.env.toObject()`